sci-libs/ccfits/ccfits-2.5.recipe

```diff
@@ -0,0 +1,114 @@
+SUMMARY="C++ Wrappers for the cfitsio library"
+DESCRIPTION="CCfits is an object oriented interface to the cfitsio library. \
+It is designed to make the capabilities of cfitsio available to programmers \
+working in C++. It is written in ANSI C++ and implemented \
+using the C++ Standard Library with namespaces, exception handling, \
+and member template functions."
+HOMEPAGE="https://heasarc.gsfc.nasa.gov/fitsio/CCfits"
+COPYRIGHT="2002-2019 Ben Dorman, Craig Gordon, Bryan Irby - HEASARC, NASA/GSFC"
```

Comment:
Maybe use the multi line copyright guideline here? https://github.com/haikuports/haikuports/wiki/HaikuPorter-BuildRecipes#a-simple-example-using-the-recommended-style-for-a-multi-line-copyright

Reply:
But they are all from the same team/university...

Reply:
I've seen in other ports. There is no a common way to do that, but I've seen here and there that multi-author lines looks ok if they belong to the same entity (for ex. zlib)